src/server_manager/web_app/app.ts

```diff
@@ -1045,6 +1036,11 @@ export class App {
       this.appRoot.showNotification(this.appRoot.localize('saved'));
       serverView.accessKeyDataLimit = dataLimitToDisplayDataAmount(limit);
       this.refreshTransferStats(this.selectedServer, serverView);
+      this.surveys
```

Comment:
See my suggestion below. This can change to
```ts
this.surveys.presentDataLimitsEnabledSurvey()
```

Reply:
Done.